### ISSUE TITLE
UI for monitoring statistics

### ISSUE THREAD

**Issue by AGWA** [SSLMate/certspotter#79]
It should be possible for users to see the current state of monitoring - e.g. which logs are being monitored, how recently they were contacted, what the backlog is

Maybe this could be provided by an optional HTTP server?

**Comment by chayleaf** [comment]
A /metrics endpoint for Prometheus would be the best option imo

**Comment by chayleaf** [comment]
it looks like it got standardized as [OpenMetrics](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md). The spec is obviously quite wordy, a short overview is available [here](https://prometheus.io/docs/instrumenting/exposition_formats/)

**Comment by vt-alt** [comment]
Prometheus metrics is good and modern way for a centralized monitoring, but just some stat into systemd journal would be as good when people don't run Prometheus.

ps. Just an example Prometheus metrics for our debuginfod instance: (how simple they looks)
https://debuginfod.altlinux.org/metrics

**Comment by schildbach** [comment]
Yes, I think some statistics printed to the log/stdout from time to time would already help a lot.

**Comment by chayleaf** [comment]
also gonna link [this](https://github.com/SSLMate/certspotter/discussions/73#discussioncomment-7360156) here

**Comment by AGWA** [comment]
certspotter v0.25.0 includes an experimental command **certspotter-metrics** which outputs Prometheus metrics.

For now, it's a separate command that you can run from cron and export using node_exporter's textfile collector, but a future version of certspotter might directly export metrics from a builtin HTTP server.

Read about it in the README: https://github.com/SSLMate/certspotter#monitoring-certspotter-with-prometheus-experimental

Here are the metrics that it outputs: https://github.com/SSLMate/certspotter/blob/master/man/certspotter-metrics.md#metrics

Feedback welcome about what metrics would be useful.
